lib/modules/manager/flux/extract.ts

```diff
@@ -14,20 +13,18 @@ import type {
 } from './types';
 
 function readManifest(content: string, file: string): FluxManifest | null {
-  if (isSystemManifest(file)) {
-    const versionMatch = regEx(
-      /#\s*Flux\s+Version:\s*(\S+)(?:\s*#\s*Components:\s*([A-Za-z,-]+))?/
-    ).exec(content);
-    if (!versionMatch) {
-      return null;
-    }
-    return {
-      kind: 'system',
-      file,
-      version: versionMatch[1],
-      components: versionMatch[2],
-    };
+  const versionMatch = regEx(
+    /#\s*Flux\s+Version:\s*(\S+)(?:\s*#\s*Components:\s*([A-Za-z,-]+))?/
+  ).exec(content);
+  if (!versionMatch) {
+    return null;
   }
+  return {
+    kind: 'system',
+    file,
+    version: versionMatch[1],
+    components: versionMatch[2],
+  };
 
   const manifest: FluxManifest = {
```

Comment:
This is now unreachable, no? This manager still needs to support non-system resources, e.g. [HelmReleases](https://docs.renovatebot.com/modules/manager/flux/#additional-information).

Reply:
Yes, I've approved tests just now because I assume it will show some regression errors